### ISSUE TITLE
Multicolor Theme: Multiple Wallpapers & MPD Previews

### ISSUE THREAD

**Issue by satheevrin** [lcpz/awesome-copycats#303]
Hey, your themes are absolutely stunning and I fell in love with multicolor the day I set it up for personal use and figured out how to customize it a bit for myself.

There are a few things I'm confused about:

First of all, I'd like to try and set two different wallpapers because I'm using two monitors instead of one. I figured out how to manipulate it enough to set one wallpaper, but I'm confused as how to set the other for a secondary screen. I've been reading documentation and doing Google searches but in the end, I thought asking you might be a better idea about this. 

Is there a way to make it so that I could have two different wallpapers in `theme.wallpaper                                 = theme.confdir .. "/wall.png"` This only sets one wallpaper as far as I know, but I'd like to have two of them, one for each screen

Unfortunately, I am still very new to the ricing world and I'm in the stages where I'm trying to learn.

Would you offer me some direction? I would appreciate it

-----------------------------

Additionally, in your screenshots, you have the MPD widget displaying album art. Mine doesn't do this even though I've figured out how to set MPD up. Am I missing something?

**Comment by jorgemota** [comment]
I am looking to get the same behavior... 3 monitors and I am unable to setup the album art :(

**Comment by lcpz** [comment]
I missed this post, sorry.

#### Multiple Wallpapers

In `theme.lua`, modify as follows:

```diff
-theme.wallpaper = theme.confdir .. "/wall.png"
+theme.wallpaper = { theme.confdir .. "/wall1.png", theme.confdir .. "/wall2.png" }
```

Then, add this function before `theme.at_screen_connect(s)`:

```lua
local function wallpaper(s)
    return theme.wallpaper[s.index - 1 if s.index and s.index > 0 or 0]
end
```

Alternatively, there is a myriad of tools you can use. The first one I found: [HydraPaper](https://hydrapaper.gabmus.org).

#### MPD Cover Art

Be sure you have a cover art image file in the album folder. Thanks @jorgemota for answering!

**Comment by satheevrin** [comment]
That's okay! Thank you for getting back to me with this. I thought it was abandoned!

This doesn't work, unfortunately....

I tried modifying that line like you said:  

[code]theme.wallpaper                                 = theme.confdir .. "/0.png", theme.confdir .. "/3.jpg"[/code]

Then I went down to [code]function theme.at_screen_connect(s)[/code]  at line 257, just before that function, I put in [code]local function wallpaper(s)
    return theme.wallpaper[s.index - 1 if s.index and s.index > 0 or 0]
end[/code]

However, this broke my theme (Spat out errors and took me back to the failsafe awesome theme) and I had to undo the changes in my editor.
